html/status.php

```diff
@@ -53,7 +61,11 @@
             foreach ($device['udp_ports'] as $port){
                 array_push($open_ports, 'UDP/'.$port);
             }
-            echo ('<tr><td>'.strtoupper($device['mac']).'</td>'.'<td>'.$device['ip'].'</td>'.'<td>'.$device['vendor'].'</td>'.'<td>'.implode(', ', $open_ports).'</td></tr>');
+            echo ('<tr style="cursor: pointer;" onclick=show_hide_row("hidden_row'.$i.'");><td>'.$device['hostname'].'</td>'.'<td>'.strtoupper($device['mac']).'</td>'.'<td>'.$device['ip'].'</td>'.'<td>'.$device['vendor'].'</td>'.'<td><a href="#" onclick=show_hide_row("hidden_row'.$i.'");">Show more</a></td></tr>');
```

Comment:
I added a small css style tag for the users to know that they can click on the rows of the table.

`style="cursor: pointer;"`